cmd/antrea-agent-simulator/simulator.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main under directory cmd parses and validates user input,
+// instantiates and initializes objects imported from pkg, and runs
+// the process.
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent"
```

Comment:
Antrea related imports should be the third block, like those in `cmd/antrea-agent-simulator/main.go`
@tnqn IIRC, there's a check but it seems not working on this file?